2023-09-13 09:04:56,526 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-13 09:05:00,925 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-13 09:05:06,711 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-13 09:05:10,141 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-13 09:05:10,365 | ERROR | OMAudit | user=hadoop | ip=10.9.0.17 | op=READ_VOLUME {volume=81712-volume-for-prepare} | ret=FAILURE | VOLUME_NOT_FOUND org.apache.hadoop.ozone.om.exceptions.OMException: Volume 81712-volume-for-prepare is not found
	at org.apache.hadoop.ozone.om.VolumeManagerImpl.getVolumeInfo(VolumeManagerImpl.java:61)
	at org.apache.hadoop.ozone.om.OzoneManager.getVolumeInfo(OzoneManager.java:2613) 
2023-09-13 09:05:13,672 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-09-13 09:05:13,897 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {prevKey=, prefix=, maxKeys=1000, username=hadoop} | ret=SUCCESS |  
2023-09-13 09:05:14,166 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {prevKey=78758-volume-for-prepare, prefix=, maxKeys=1000, username=hadoop} | ret=SUCCESS |  
